src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,27 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(SessionFactory sessionFactory) {
+        actorDao = new ActorDaoImpl(HibernateUtil.getSessionFactory());
```

Comment:
use constructor variable 
```suggestion
        actorDao = new ActorDaoImpl(sessionFactory);
```